presto-hive/src/test/java/com/facebook/presto/hive/TestHiveCanonicalPlanHashes.java

```diff
@@ -58,7 +62,19 @@ public class TestHiveCanonicalPlanHashes
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return HiveQueryRunner.createQueryRunner(ImmutableList.of(ORDERS, LINE_ITEM));
+        QueryRunner queryRunner = HiveQueryRunner.createQueryRunner(ImmutableList.of(ORDERS, LINE_ITEM));
+        if (queryRunner.getStatsCalculator() instanceof HistoryBasedPlanStatisticsCalculator) {
+            ((HistoryBasedPlanStatisticsCalculator) queryRunner.getStatsCalculator()).setPrefetchForAllPlanNodes(true);
```

Comment:
In unit tests, we want to test for queries which do not have join/aggregation too.